example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Thu Dec 08 21:41:12 EET 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.4-all.zip
```

Comment:
Similarly, compared with tipsi-stripe project. So, what do you think, which one we should use?